src/app/stream-deck/async-module.ts

```diff
@@ -74,8 +74,9 @@ function streamDeckSelectItem(item: DimItem): ThunkResult {
             selection: {
               label: item.name,
               subtitle: item.typeName,
-              item: item.id,
+              item: item.index.replace(/-.*/, ''),
```

Comment:
for inventory items that don't have an instanceId (I mean materials, or other stackable items) since I noticed their share an initial piece of their index (before a '-') and in this case I want the generic item, not a specific stack. I didn't found a better alternative.

Reply:
In that case the initial part is their item hash